examples/jupyter/requirements.txt

```diff
@@ -4,5 +4,5 @@ matplotlib>=3.2
 networkx>=2.4
 numpy>=1.18
 pandas>=1.0
-pyarrow>=0.17,<5.0
+pyarrow>=0.17
```

Comment:
Would it make sense to provide a tight instead of just a lower bound? Explicitly ratcheting up a volatile dependency seems prudent, given that we've been bitten several times with this in the past.

Reply:
We should fix this managing the dependencies with [poetry](https://python-poetry.org/) instead.
Related to https://github.com/tenzir/vast/issues/1779.